Remotion/Validation/UnitTests/Implementation/DiagnosticInformationValidatorFormatterDecoratorTest.cs

```diff
@@ -81,17 +81,17 @@ public void Format_NotEmptyOrWhitespaceValidator ()
     }
 
     [Test]
-    public void Format_ILengthValidators ()
+    public void Format_IMinimumLengthValidatorAndIMaximumLengthValidator ()
     {
       var validator1 = new LengthValidator(5, 10, new InvariantValidationMessage("Fake Message"));
       var validator2 = new MaximumLengthValidator(12, new InvariantValidationMessage("Fake Message"));
       var validator3 = new MinimumLengthValidator(2, new InvariantValidationMessage("Fake Message"));
       var validator4 = new ExactLengthValidator(4, new InvariantValidationMessage("Fake Message"));
 
       Assert.That(_formatter.Format(validator1, _typeNameFormatter), Is.EqualTo("LengthValidator { MinLength = '5', MaxLength = '10' }"));
-      Assert.That(_formatter.Format(validator2, _typeNameFormatter), Is.EqualTo("MaximumLengthValidator { MinLength = '0', MaxLength = '12' }"));
-      Assert.That(_formatter.Format(validator3, _typeNameFormatter), Is.EqualTo("MinimumLengthValidator { MinLength = '2', MaxLength = '' }"));
-      Assert.That(_formatter.Format(validator4, _typeNameFormatter), Is.EqualTo("ExactLengthValidator { MinLength = '4', MaxLength = '4' }"));
+      Assert.That(_formatter.Format(validator2, _typeNameFormatter), Is.EqualTo("MaximumLengthValidator { MaxLength = '12' }"));
+      Assert.That(_formatter.Format(validator3, _typeNameFormatter), Is.EqualTo("MinimumLengthValidator { MinLength = '2' }"));
+      Assert.That(_formatter.Format(validator4, _typeNameFormatter), Is.EqualTo("ExactLengthValidator { Length = '4' }"));
     }
```

Comment:
Please split this into 4 separate tests. see the other cases. it was just done that way before because they used the same interface / baseclass during detection. that's no longer the case.